src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -40,7 +40,7 @@ class OrchestratorCli(orchestrator.OrchestratorClientMixin, MgrModule):
             "perm": "r"
         },
         {
-            'cmd': "orchestrator osd add "
+            'cmd': "orchestrator osd create "
```

Comment:
Once and for all: `orchestrator osd create` or `orchestrator osd add`?

Reply:
We are using "osd rm" so... i vote for "osd add"!